tests/backends/base/test_base.py

```diff
@@ -297,3 +315,21 @@ def test_set_autocommit_health_checks_enabled(self):
         connection.commit()
         connection.set_autocommit(True)
         self.assertIs(new_connection, connection.connection)
+
+
+class MultiDatabaseTests(TransactionTestCase):
+    available_apps = []
+    databases = {"default", "other"}
+
+    def test_multi_database_init_connection_state_called_once(self):
+        for db in self.databases:
+            with self.subTest(database=db):
+                with patch.object(
+                    connections[db],
+                    "check_database_version_supported",
+                ) as mocked_check_database_version_supported:
+                    connections[db].init_connection_state()
+                    connections[db].init_connection_state()
+                self.assertTrue(
+                    mocked_check_database_version_supported.assert_called_once
+                )
```

Comment:
This is always truthy:
```suggestion
                mocked_check_database_version_supported.assert_called_once()
```

Reply:
Fixed.